cdap-ui/app/cdap/components/Pagination/Pagination.less

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+.pagination-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow-x: hidden;
+
+  .pagination-content {
+    display: inline-block;
+    width: 100%;
+  }
+
+  .change-page-panel {
+    display: inline-block;
+    position: absolute;
+    cursor: pointer;
+
+    /*Page height minus footer height*/
+    height: ~"-moz-calc(100vh - 50px)";
```

Comment:
This should just be `100%`. It should take the height of the parent (which is the `.pagination-container` which should have appropriate height.